openwisp_utils/ci.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file contains functions that are required by
+multiple openwisp repositories for testing
+during continuous integration
+"""
+import os
+import re
+
+
+def CheckMigrationName():
+    """
+    Check if the migration files created have a descriptive
+    name; if default name pattern is found, exit with 1
+    """
+    MIGRATION_PATH = os.environ["MIGRATION_PATH"]
+    PROJECT_NAME = os.environ["TRAVIS_REPO_SLUG"]
```

Comment:
since this is a function you don't need uppercase variable names (which is a convention to mean constants and global constants), you should use all lowercase variable names

Reply:
Oops, i started this task with a shell script and copied all the uppercase variables from there! 
I am on fixing this! :smile: